tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,62 @@
+# Ensure that COS Lite deploys successfully, the
+# relations are setup as expected and we can access the dashboards.
+run_deploy_coslite() {
+	echo
+
+	local model_name file overlay_path admin_passwd
+	model_name="deploy-coslite"
+	file="${TEST_DIR}/${model_name}.log"
+	ensure "${model_name}" "${file}"
+
+	IPADDR=$(ip -4 -j route | jq -r '.[] | select(.dst | contains("default")) | .prefsrc')
+	# Some wifi devices do not set prefsrc ip. So in its place just have the local ip address as the range.
+	if [ "${IPADDR}" = "null" ]; then
+		IPADDR=$(hostname -I | awk '{print $1}')
+	fi
+
+	microk8s enable metallb:"$IPADDR"-"$IPADDR"
+	microk8s kubectl rollout status deployments/hostpath-provisioner -n kube-system -w
+	microk8s kubectl rollout status deployments/coredns -n kube-system -w
+	microk8s kubectl rollout status daemonset.apps/speaker -n metallb-system -w
+
+	overlay_path="./tests/suites/coslite/overlay"
+	juju deploy cos-lite --trust --channel=edge --overlay "${overlay_path}/offers-overlay.yaml" --overlay "${overlay_path}/storage-small-overlay.yaml"
+
+	wait_for "active" '.applications["alertmanager"] | ."application-status".current'
+	wait_for "active" '.applications["loki"] | ."application-status".current'
+	wait_for "active" '.applications["prometheus"] | ."application-status".current'
+	wait_for "active" '.applications["traefik"] | ."application-status".current'
+	wait_for "active" '.applications["grafana"] | ."application-status".current'
+
+	# check for relations
+	wait_for 5 '.applications["alertmanager"] | ."relations" | length'
+	wait_for 3 '.applications["grafana"] | ."relations" | length' 1800
+	wait_for 5 '.applications["loki"] | ."relations" | length'
+	wait_for 6 '.applications["prometheus"] | ."relations" | length'
+	wait_for 2 '.applications["traefik"] | ."relations" | length' 1800
```

Comment:
Why a different timeout here and line 33?

Reply:
I'm a little surprised these need non-default timeouts at all. Really, if all your applications are active, you shouldn't need to wait any time for the relations to come up right?

Reply:
Oh shoot. Probably I had it mistaken for something else. Nice catch. Thanks.

Reply:
EDIT: I think you might be correct. I got confused here sometimes the traefik stalls installing the agent giving me some mixed conclusions.